password_security/models/res_users.py

```diff
@@ -156,3 +156,20 @@ def _set_encrypted_password(self, encrypted):
                 'password_crypt': encrypted,
             })],
         })
+
+    # Dirty fix because this was failing when any theme was installed
+    # I think this should be a fix in the core.
+
+    @api.multi
+    def _is_system(self):
+        if not self:
+            return False
+        return super(ResUsers, self)._is_system()
+
+    @api.multi
+    def _is_superuser(self):
+        if not self:
+            return False
+        return super(ResUsers, self)._is_superuser()
+
+    ############ End of dirty fix ###############
```

Comment:
This is not a correct solution odoo/odoo@679d278 @moylop,the variable assignation is incorrect, this is fixed on master as I am showing.

I did this solution and it is working properly, the problem is the inheritance of this module not odoo itself. Odoo is Ok!